admin/class-yoast-dashboard-widget.php

```diff
@@ -38,6 +38,7 @@ public function __construct( WPSEO_Statistics $statistics = null ) {
 	 * Adds dashboard widget to WordPress
 	 */
 	public function add_dashboard_widget() {
+		add_filter( 'postbox_classes_dashboard_wpseo-dashboard-overview', array( $this, 'wpseo_dashboard_overview_class' ) );
```

Comment:
@afercia I must have been very confused when commenting. This is completely fine.